packages/onboarding/src/step-container-v2/components/ContentWrapper/ContentWrapper.tsx

```diff
@@ -3,22 +3,26 @@ import { ReactNode } from 'react';
 
 import './style.scss';
 
+type Width =
+	| 'wide'
+	| 'huge'
+	| 'huge-plus' // Non-standard width introduced for plans step
```

Comment:
This is what the plans step looks like if it uses the standard `huge` breakpoint for the max width

![CleanShot 2025-04-01 at 17 50 41@2x](https://github.com/user-attachments/assets/95d8b9dc-8172-46eb-aadc-68fc3cb1d73e)

This is what it looks like in production. Subtle, but slightly more breathing space.

![CleanShot 2025-04-01 at 17 54 17@2x](https://github.com/user-attachments/assets/051f0692-77aa-4347-bc66-8f67ca69ddbf)

Maybe the inconsistency isn't worth it.

@nuriapenya @fcoveram

Reply:
non-blocking, in terms of naming could be more conventional. maybe `wide` `x-wide` , `2x-wide` or something that the brain might automatically guess the next option. I wouldn't guess `wide` to `huge`

Reply:
> I wouldn't guess `wide` to `huge`

😅 yeah same. I went with this names to be consistent with the "standard" Gutenberg breakpoints. Core first?